crates/db_schema/src/source/post.rs

```diff
@@ -36,11 +36,8 @@ pub struct Post {
 #[cfg_attr(feature = "full", derive(Insertable, AsChangeset))]
 #[cfg_attr(feature = "full", diesel(table_name = post))]
 pub struct PostInsertForm {
-  #[builder(!default)]
   pub name: String,
-  #[builder(!default)]
   pub creator_id: PersonId,
-  #[builder(!default)]
```

Comment:
These shouldn't be removed, because the database needs them. Would be better to handle the missings in the apub code.

Reply:
Fixed